src/pages/admin/donations/campaign/[id].tsx

```diff
@@ -0,0 +1,10 @@
+import DonationsPage from 'components/donations/DonationsPage'
+import { securedAdminProps } from 'middleware/auth/securedProps'
+import { endpoints } from 'service/apiEndpoints'
+
+export const getServerSideProps = securedAdminProps(
+  ['common', 'auth', 'admin', 'donations', 'validation'],
+  (ctx) => endpoints.campaign.getDonations(ctx.query.id as string).url,
+)
+
+export default DonationsPage
```

Comment:
Is there any reason for defining a new page or we can use the existing DonationPage and just filter the Donations with query param?
The initial idea was the campaign to come through a query param in this way: /admin/donations?campaignId=[uuid] while this implementation is creating a new route admin/donations/campaign/[id]